mediacapture-streams/MediaStream-MediaElement-preload-none-src.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!-- Copyright © 2016 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
```

Comment:
Yes - per separate discussion with w3c folks and internal Google, upstreamed versions of Chromium blink layout tests like this one will need this stamp. I too like minimalism FWIW :)